lib/grunt/task.js

```diff
@@ -1,5 +1,11 @@
 'use strict';
 
+// Keep track of the number of log.error() calls.
+var errorcount,
+
+  // The last specified tasks message.
+  lastInfo;
```

Comment:
Are you saying ESLint doesn't allow multiple var statements?

@cowboy has [some good opinions](http://benalman.com/news/2012/05/multiple-var-statements-javascript/) on that. So Grunt would definitely prefer to use multiple var statements.